### ISSUE TITLE
Machine Learning Preprocessing tutorial innacuracies

### ISSUE THREAD

**Issue by gully** [KeplerGO/lightkurve#642]
The new [machine learning pre-processing tutorial](https://docs.lightkurve.org/tutorials/05-advanced_patterns_binning.html) departs from the Shallue and Vanderburg preprocessing in two ways.  First, the normalization is off because of my own failing--- the normalization is [-1, 0], not [-1, 1].  Second, and more subtly, the local view binning in Shallue & Vanderburg conducts an operation akin to smoothing: each bin in the 201 sample view sums values from nearby native samples beyond its bin boundaries, where nearby is dictated in the results section of their paper.  

Basically this binning choice acts as a smoothing or kernel density estimate, where the kernel is a box function larger than the bin size.  I'm not sure we would want to alter lightkurve to match this behavior, but we should update the tutorial to alert users of the difference, so as not to mislead them that the mapping is exactly reproducing the published work.

While I'm listing departures from the paper, the flattening procedure in the tutorial is Sav-Gol, while the published work uses splines.

**Comment by barentsen** [comment]
I'm tempted to close this issue because I don't think it's the end of the world that the tutorial doesn't *exactly* replicate Shallue & Vanderburg; however if someone opened a PR to refine the tutorial at any point in the future, I'd be happy to merge it in a heartbeat!
